Makefile

```diff
@@ -0,0 +1,26 @@
+.PHONY:
+	install-dev
+	install
+	lint
+	format
+	run
+
+install-dev: install
+	pip install -e .
```

Comment:
I think this should be `pip install -e ".[dev]"

Reply:
[b1e4b1c](https://github.com/moradology/impact-explorer/pull/1/commits/b1e4b1c2d07a52eb1644459069de9f3d6a811aa0)